test/functional/Java11andUp/src/org/openj9/test/java_lang/Test_String.java

```diff
@@ -63,4 +63,41 @@ public void testValueOfCodePoint() throws IllegalAccessException, IllegalArgumen
 			// Expected exception
 		}
 	}
-}
\ No newline at end of file
+
+	/*
+	 * Test Java 11 method String.repeat(count)
+	 */
+	@Test(groups = { "level.sanity" })
+	public void testRepeat() throws IllegalAccessException, IllegalArgumentException, InvocationTargetException, NoSuchMethodException, SecurityException {
```

Comment:
Seems like this was copied from somewhere, not sure any of these exceptions need to be declared for this test.